reactivesocket/src/main/scala/akka/stream/alpakka/reactivesocket/scaladsl/HelloWorld.scala

```diff
@@ -0,0 +1,73 @@
+package akka.stream.alpakka.reactivesocket.scaladsl
+
+import io.reactivesocket.transport.tcp.server.TcpTransportServer
+import io.reactivesocket.server.ReactiveSocketServer
+import io.reactivesocket.server.ReactiveSocketServer.SocketAcceptor
+import io.reactivesocket.ConnectionSetupPayload
+import io.reactivesocket.lease.LeaseEnforcingSocket
+import io.reactivesocket.ReactiveSocket
+import io.reactivesocket.lease.DisabledLeaseAcceptingSocket
+import io.reactivesocket.AbstractReactiveSocket
+import org.reactivestreams.Publisher
+import io.reactivesocket.Payload
+import io.reactivesocket.client.ReactiveSocketClient
+import io.reactivesocket.transport.tcp.client.TcpTransportClient
+
+import io.reactivesocket.client.KeepAliveProvider._
+import io.reactivesocket.client.SetupProvider._
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import akka.stream.scaladsl.Source
+import akka.stream.scaladsl.Sink
+import io.reactivesocket.util.PayloadImpl
+import scala.concurrent.Future
+import io.reactivesocket.frame.ByteBufferUtil
+
+object HelloWorld {
```

Comment:
This corresponds to https://github.com/ReactiveSocket/reactivesocket-java/blob/0.5.x/reactivesocket-examples/src/main/java/io/reactivesocket/examples/transport/tcp/requestresponse/HelloWorldClient.java